### PR TITLE
Fix appsecret_proof encoding

### PR DESCRIPTION
**Pull request by damianfral**

### CODE REVIEW

**Comment by damianfral** [comment]
hey @psibi, I made a mistake in the implementatioh of makeAppSecretProof, forgot to do bas64 encoding. Here is the PR to fix that. :)

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# Version 2.0.1
```
Comment (psibi) [inline CHANGELOG.md]: Note that 2.0.0 isn't released yet, so, you can put the changelog under that.

File: package.yaml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 name: fb
-version: '2.0.0'
+version: '2.0.1'
```
Comment (psibi) [inline package.yaml]: This doesn't need to be changed.

**Comment by damianfral** [comment]
@psibi, done. :)

**Comment by psibi** [comment]
Thanks!

**Pull request merged**